docs/kratos/social-signin/09_ory.mdx

```diff
@@ -0,0 +1,182 @@
+---
+id: ory
+title: Add Ory OAuth2 or Ory Hydra as a social sign-in provider
+sidebar_label: Ory
+---
+
+# Ory
+
+This **Ory** option allows you to add Ory OAuth2 as OIDC provider. In Ory Network, you can create this provider using the Ory
+Console or the Ory CLI. When self-hosting Ory, deploy the Ory Hydra Federation server and use the `hydra` CLI to create the
+client.
+
+The following snippet shows how to create it using the CLI:
+
+```bash
+ory create project --name "OAuth2 Server - Example Corp"
+# Note down the project ID
+export project_id=your-project-id # replace with your project ID
+
+ory create oauth2-client --project $project_id \
+ --name "Example Corp" \
+ --grant-type authorization_code,refresh_token \
+ --response-type code \
+ --scope openid,offline_access,email \
+ --redirect-uri https://your-project-slug.projects.oryapis.com/self-service/methods/oidc/callback/H1o_k--i # replace with your redirect URI
+```
+
+If you want to learn more head over to the [Ory OAuth2](../../oauth2-oidc/index.md) documentation.
+
+To add your Ory OAuth2 server as a social sign-in provider, you need these details:
```

Comment:
maybe add the context - the fact that we'll set up the connection _just like a sign-in with Google_, which works against our Ory SSO provider using the "generic" adapter, because Ory implements the spec correctly on both ends :)